### ISSUE TITLE
The /test alone bloats up this module's size by over 10000%

### ISSUE THREAD

**Issue by CamiloMM** [PaddeK/node-maxmind-db#13]
Having such a huge-ass `.mmdb` file in the `/test` dir bloats the package from < 300**kb** to > 29**mb**. That's **over 10000% increase in size** for something that *virtually nobody will ever use*.

Please think of the children and download this file on-demand for tests (MaxMind doesn't change the URL so it would be easy).

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;![he can't bear it](https://cloud.githubusercontent.com/assets/2404138/7900531/c2b01a78-0734-11e5-8a2e-8daf35f4a4f4.jpg "he can't bear it")

Do you see that polar bear? He's sad because his rainforest is melting, his ozone layer is being fracked, he's almost ran out of Coca-Cola and his hard-drive is full of bloated NPM modules, *and it's all your fault*.

How can he save that naughty asian panda porn now? :( Please fix. For the pandas. For the future.

**Comment by oschwald** [comment]
FWIW, Maxmind also provides [small test databases](https://github.com/maxmind/MaxMind-DB/tree/master/test-data).

**Comment by CamiloMM** [comment]
That's neat, but they're still several megabytes in size (`GeoIP2-City-Test.mmdb` is 19mb).

**Comment by oschwald** [comment]
I think you are misreading the file size. That file is 19 kb in size.

**Comment by CamiloMM** [comment]
You're right, I read "19.093 kb" as "19,093 kb". That size is perfectly fine to publish on NPM!

**Comment by behcet** [comment]
Please consider using [.npmignore](https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package) to exclude files when publishing to npm

**Comment by knoxcard** [comment]
https://github.com/runk/node-maxmind
